src/App.scss

```diff
@@ -0,0 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.App {
+  text-align: center;
```

Comment:
Check older comment -
classes should start from lowercase
things like that should be done in files like normalize.scss, maybe at reset.scss or even general.scss, but most likely (as you found that it's better to render components inside App.js than pure jsx code) you may want to style those components instead of App.js

//new
In general I meant that you should avoid class .App or you should avoid styling whole App component